packages/addon/public/token-bridge.js

```diff
@@ -62,6 +63,12 @@ window.addEventListener('message', (e) => {
     }
   }
 
+  if (e?.data?.type === SIGN_IN_COMPLETE) {
+    browser.runtime.sendMessage({
+      type: SIGN_IN_COMPLETE,
+    });
+  }
+
```

Comment:
Closing the tab after authentication, part 2/5

The bridge receives the "ok to close" message from the web app ( `ClosePage.vue`)

It then sends the same message to `background.ts` using extension messaging.